xrdsst/controllers/service.py

```diff
@@ -327,6 +374,32 @@ def remote_update_service_parameter(ss_api_config, service_description_conf, ser
         except ApiException as err:
             BaseController.log_api_error('ServicesApi->update_service', err)
 
+    def remote_list_service_descriptions(self, ss_api_config, security_server, client):
+        clients_api = ClientsApi(ApiClient(ss_api_config))
+        try:
+            service_descriptions_list = []
+            render_data = []
+            client_ids = client.split(',')
```

Comment:
i've created in another branch a this function in util.py:
def parse_argument_list(arg):
    list = arg.split(',')
    return list
    
It's just s simple split but maybe it's better to have it centralize in one single place, if not, if in the future we want to change the character separator we will need to find it through all the code

Reply:
I added the same functionality and used it